src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/ReindexJobTask.cs

```diff
@@ -511,6 +511,12 @@ private async Task<ReindexJobQueryStatus> ProcessQueryAsync(ReindexJobQueryStatu
 
                     if (query.FailureCount >= _reindexJobConfiguration.ConsecutiveFailuresThreshold)
                     {
+                        var issue = new OperationOutcomeIssue(
+                            OperationOutcomeConstants.IssueSeverity.Error,
+                            OperationOutcomeConstants.IssueType.Exception,
+                            ex.Message);
```

Comment:
>ex [](http://example.com/codeflow?start=28&length=2)

If it was FhirException I think it would be ok to push it to user, but this is generic exception. 

What user suppose to do with it?

Reply:
Good point. I am currently using this to pass an action item to the user on `PreconditionFailed` exceptions, but I can see how this might lead to us sending info to the user that wouldn't be helpful. Working on finding a nice fix now.

Reply:
Added an extra catch block in 52ea697.